### ISSUE TITLE
v5 Breaks on Python 3.8 due to type annotation syntax

### ISSUE THREAD

**Issue by Kytech** [r0x0r/pywebview#1324]
### Specification
- pywebview version: v5.0 - latest (v5.0.3 at present) are all affected
- operating system: Windows
- web renderer: Not Applicable for this bug - see description

### Description

Simply importing the pywebview package on python 3.8 fails due to use of type hint syntax that is not supported in python 3.8. A particular example of the unsupported syntax is detailed in the error message below where `list['Element']` is used as a type hint inside this package, which is not supported on python 3.8 and would need to be specified instead as `List[Element]` using python's `List` type from the `typing` package to properly support python 3.8.

```python
import webview
```

```
Traceback (most recent call last):
  File "main.py", line 2, in <module>
    import webview
  File "C:\Users\kytec\miniconda3\envs\thesis\lib\site-packages\webview\__init__.py", line 34, in <module>
    from webview.window import Window
  File "C:\Users\kytec\miniconda3\envs\thesis\lib\site-packages\webview\window.py", line 20, in <module>
    from webview.dom.dom import DOM
  File "C:\Users\kytec\miniconda3\envs\thesis\lib\site-packages\webview\dom\dom.py", line 3, in <module>
    from webview.dom.element import Element
  File "C:\Users\kytec\miniconda3\envs\thesis\lib\site-packages\webview\dom\element.py", line 47, in <module>
    class Element:
  File "C:\Users\kytec\miniconda3\envs\thesis\lib\site-packages\webview\dom\element.py", line 197, in Element
    def children(self) -> list['Element']:
TypeError: 'type' object is not subscriptable
```

I discovered the issue while using python 3.8.10, though it will likely impact all versions prior to python 3.9, which added support for the type hint syntax that is causing this issue. This package presently indicates support for python 3.7 and greater in its `pyproject.toml` file and in the published package on PyPI, so pip installing pywebview on python 3.8 results in downloading one of the v5 versions, which will fail with the above error message.

The likely fix is to either adjust the type hint syntax to a variant supported on python 3.8, or to remove python versions before 3.9 as supported versions for all of the future and existing v5.0 packages so that a pip install on these versions of python does not result in a broken package.

### Practicalities

- I am willing to work on this issue myself, though I cannot make this a priority any time soon at the moment due to other obligations, though I wanted to bring attention to this as it may warrant a fix before I could get around to drawing up a PR as it results in completely unusable packages being installed on certain python versions.

- I am NOT prepared to support this issue financially.

**Comment by r0x0r** [comment]
Thanks for bringing this up to my attention. Type signatures are fixed in master now.

**Comment by Kytech** [comment]
Just tested the package from master and it loads up on python3.8

**Comment by r0x0r** [comment]
Released in 5.0.4
